src/config/lang/english.go

```diff
@@ -305,6 +305,9 @@ const (
 	CmdPackageClusterSourceFallback = "%q does not satisfy any current sources, assuming it is a package deployed to a cluster"
 	CmdPackageInvalidSource         = "Unable to identify source from %q: %s"
 
+	//zarf lint
+	CmdLintShort = "Tools for validating schema / ensuring best practices"
```

Comment:
ensuring best practices, should be like schema is correct, and identifies deprecations and warns the user.